crates/egui/src/widgets/slider.rs

```diff
@@ -72,6 +73,8 @@ pub struct Slider<'a> {
     spec: SliderSpec,
     clamp_to_range: bool,
     smart_aim: bool,
+    smart_aim_values: Vec<f64>,
+    snap_radius: f32,
```

Comment:
I think it's better to just use the normal `aim_radius`

```suggestion
```

Reply:
If only using aim_radius, the snapping will behave something like this:
![10](https://user-images.githubusercontent.com/19912588/214060086-388b8371-597e-46f6-bc04-bafa679df972.gif)

But do we not also want the flexibility to support this kind of snapping? And to be able to decide for an individual slider what kind of granularity between snap values is supported? That's what I was thinking with having a separate `snap_radius` for each slider.
![3](https://user-images.githubusercontent.com/19912588/214060242-6d75927e-11b9-402a-8c33-b8126f4c9eef.gif)

Reply:
Ah, good point. I think these should be two separate modes. One is "snap if close" the other is "only use snap-values".

Reply:
Okay, I added that with a `snap_values_only` bool in `Slider`